common/src/main/java/com/wynntils/screens/maps/SeaskipperMapScreen.java

```diff
@@ -0,0 +1,431 @@
+                            TextShadow.OUTLINE);
+
+            renderYOffset += 10;
+
+            FontRenderer.getInstance()
+                    .renderText(
+                            poseStack,
+                            CodedString.fromString("Click to go here!"),
+                            10,
+                            10 + renderYOffset,
+                            CommonColors.BLUE,
+                            HorizontalAlignment.LEFT,
+                            VerticalAlignment.TOP,
+                            TextShadow.OUTLINE);
+        } else if (origin) {
+            renderYOffset += 20;
+
+            FontRenderer.getInstance()
+                    .renderText(
+                            poseStack,
+                            CodedString.fromString("Origin"),
+                            10,
+                            10 + renderYOffset,
+                            CommonColors.RED,
+                            HorizontalAlignment.LEFT,
+                            VerticalAlignment.TOP,
+                            TextShadow.OUTLINE);
+        } else {
+            renderYOffset += 20;
+
+            FontRenderer.getInstance()
+                    .renderText(
+                            poseStack,
+                            CodedString.fromString("Inaccessible"),
+                            10,
+                            10 + renderYOffset,
+                            CommonColors.GRAY,
+                            HorizontalAlignment.LEFT,
+                            VerticalAlignment.TOP,
+                            TextShadow.OUTLINE);
+        }
+
+        FontRenderer.getInstance()
+                .renderAlignedTextInBox(
+                        poseStack,
+                        CodedString.fromString(hoveredPoi.getName()),
+                        7,
+                        textureWidth,
+                        Texture.TERRITORY_TOOLTIP_TOP.height() + centerHeight,
+                        Texture.TERRITORY_TOOLTIP_TOP.height() + centerHeight + Texture.TERRITORY_NAME_BOX.height(),
+                        0,
+                        CommonColors.WHITE,
+                        HorizontalAlignment.LEFT,
+                        VerticalAlignment.MIDDLE,
+                        TextShadow.OUTLINE);
+
+        poseStack.popPose();
+    }
+
+    @Override
+    public boolean doMouseClicked(double mouseX, double mouseY, int button) {
+        for (GuiEventListener child : children()) {
+            if (child.isMouseOver(mouseX, mouseY)) {
+                child.mouseClicked(mouseX, mouseY, button);
+                return true;
+            }
+        }
+
+        for (SeaskipperPoi poi : Models.Seaskipper.getSeaskipperPois()) {
+            if (poi.isSelected(mouseX, mouseY)) {
+                Models.Seaskipper.buyPass(poi.getName());
+
+                return true;
+            }
+        }
+
+        return mouseClicked(mouseX, mouseY, button);
```

Comment:
This is a bad merge. I made this mistake too :)

```suggestion
        return super.doMouseClicked(mouseX, mouseY, button);
```